Kitodo/src/main/java/de/sub/goobi/metadaten/MetadatenVerifizierung.java

```diff
@@ -45,6 +47,7 @@
     List<DocStruct> docStructsOhneSeiten;
     Process myProcess;
     boolean autoSave = false;
+    private static final Logger logger = LogManager.getLogger(MetadatenHelper.class);
```

Comment:
This class is called `MetadatenVerifizierung` but you are creating a logger for class `MetadatenHelper`.